.github/workflows/linux.yml

```diff
@@ -225,8 +225,27 @@ jobs:
           mkdir -p Notes/usr/share/icons/hicolor
           cp -r ../packaging/linux/common/icons/* Notes/usr/share/icons/hicolor
           export VERSION='${{ steps.vars.outputs.version }}'
-          ./linuxdeploy-x86_64.AppImage --appdir Notes --output appimage --plugin qt
-          mv Notes*.AppImage '${{ steps.vars.outputs.file_name }}'
+          ./linuxdeploy-x86_64.AppImage --appdir Notes --plugin qt
+
+      - name: Remove unnecessary Qt plugins and libraries
+        shell: bash
+        run: |
+          set -x
+          set -e
+          cd build/Notes
+          if [[ '${{ matrix.qt-version }}' == 5.* ]]
```

Comment:
Couldn't we move this if statement to the step itself, something like this:


```yaml
name: Remove unnecessary Qt plugins and libraries
if: matrix.qt-version == 5.*
```

Not sure if this exact code works, but something like this would make the step be skipped altogether on Qt6 builds. Same thing with the macOS and Windows builds.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif

Reply:
Yes, GitHub allows us to use [`startsWith()`](https://docs.github.com/en/actions/learn-github-actions/expressions#startswith). However, [I plan](https://github.com/guihkx/notes/commit/c39a073d7aa358990196e85aa41634624068ce39) on also removing non-SQLite plugins, and those are present on both Qt 5 and Qt 6 builds, so this step will end up having to run on both builds anyway...